Program.cs

```diff
@@ -1,18 +1,43 @@
 ﻿using MongoDB.Driver;
 using MongoDB.Bson;
+using System.Diagnostics;
 
 public static class Example
 {
     public static void Main(string[] args)
     {
-        var client = new MongoClient(args[0].ToString());
+        var connectionUri = args[0].ToString();
+        var settings = MongoClientSettings.FromConnectionString(connectionUri);
+        var client = new MongoClient(settings);
 
         IMongoDatabase db = client.GetDatabase("test");
         var command = new BsonDocument { { "ping", 1 } };
         var res = db.RunCommand<BsonDocument>(command);
+        Console.WriteLine(res);
 
+        command = new BsonDocument { { "dropDatabase", 1 } };
+        res = db.RunCommand<BsonDocument>(command);
         Console.WriteLine(res);
 
+        var documentList = new List<BsonDocument>();
+        for (int i = 1; i < 5; i++)
+        {
+            var document = new BsonDocument{ {"_id", i }, { "a", i } };
+            documentList.Add(document);
+        }
+
+        // insert documents
+        var collection = db.GetCollection<BsonDocument> ("foo");
+        collection.InsertMany(documentList);
+
+        // find the document
+        var filter = Builders<BsonDocument>.Filter.Eq("a", 4);
+        var resList = collection.Find(filter).ToList();
+        BsonDocument actual = resList.Last(); 
+
+        var expected = new BsonDocument { { "_id", 4 }, { "a", 4 } };
+        Debug.Assert(expected == actual, " Value should be 4");

```

Comment:
What is the purpose of `Debug.Assert`? Does it fail CI if this fails?

I'm asking because do assertion for `Find` value, but we didn't do assertion for `ping` `dropDatabase` and `insertMany`. I think it's a bit inconsistent. :thinking:

Reply:
Yes it will fail on CI if `Debug.Assert` returns a non-zero exit code. I will use assertions for other commands too.

Reply:
Thanks, that's great :hugs: